public/app/plugins/datasource/mysql/sql_part.ts

```diff
@@ -1,86 +0,0 @@
-import { SqlPartDef, SqlPart } from 'app/angular/components/sql_part/sql_part';
-
-const index: any[] = [];
-
-function createPart(part: any): any {
-  const def = index[part.type];
-  if (!def) {
-    return null;
-  }
-
-  return new SqlPart(part, def);
-}
-
-function register(options: any) {
-  index[options.type] = new SqlPartDef(options);
-}
-
-register({
-  type: 'column',
-  style: 'label',
-  params: [{ type: 'column', dynamicLookup: true }],
-  defaultParams: ['value'],
-});
-
-register({
-  type: 'expression',
-  style: 'expression',
-  label: 'Expr:',
-  params: [
-    { name: 'left', type: 'string', dynamicLookup: true },
-    { name: 'op', type: 'string', dynamicLookup: true },
-    { name: 'right', type: 'string', dynamicLookup: true },
-  ],
-  defaultParams: ['value', '=', 'value'],
-});
-
-register({
-  type: 'macro',
-  style: 'label',
-  label: 'Macro:',
-  params: [],
-  defaultParams: [],
-});
-
-register({
-  type: 'aggregate',
-  style: 'label',
-  params: [
-    {
-      name: 'name',
-      type: 'string',
-      options: ['avg', 'count', 'min', 'max', 'sum', 'stddev', 'variance'],
```

Comment:
We are missing some of the aggregate functions from here. Is this something that we will do later, to make it customisable?

Reply:
@zoltanbedi should be fixed now.  Added a "functions" prop to the db so you will need to implement that in mssql datasource.